src/sass/_globals.scss

```diff
@@ -0,0 +1,26 @@
+* {
+    box-sizing: border-box;
```

Comment:
box-sizing: border-box możesz usunąć, bo wstawię w heada normalizer, a wnim jest zapis o tym, żeby to był border-box
za to dodaj margin: 0 i padding: 0